### ISSUE TITLE
react-test-renderer types missing shallow - ShallowRenderer

### ISSUE THREAD

**Issue by eddedd88** [flowtype/flow-typed#2352]
The types are missing this: https://reactjs.org/docs/shallow-renderer.html

So when I do `import ShallowRenderer from 'react-test-renderer/shallow`, it's unable to resolve the path.

I tried porting these missing types from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-test-renderer/shallow/index.d.ts, but I got stuck and need some guidance.

Here is what I have so far:
```
declare module "react-test-renderer/shallow" {
  declare export default class ShallowRenderer {
    static createRenderer(): ShallowRenderer;
    getMountedInstance(): ReactTestInstance;
    getRenderOutput(): React$Element<any>;
    render(element: React$Element<any>, context?: any): void;
    unmount(): void;

    // not sure how to port the following to flow
    // getRenderOutput<E extends ReactElement<any>>(): E;
  }
}
```
Can somebody help me porting this `getRenderOutput<E extends ReactElement<any>>(): E;` to flow

**Comment by AndrewSouthpaw** [comment]
Does the `getRenderOutput(): React$Element<any>` above it not work to catch all cases?

**Comment by eddedd88** [comment]
Thanks for the help, I think that should work.

I'll create a PR in the next few days

**Comment by eddedd88** [comment]
this was addressed in #2373
